sqlelf/elf.py

```diff
@@ -233,6 +238,8 @@ def mode(binary: lief_ext.Binary) -> int:
 def arch(binary: lief_ext.Binary) -> int:
     if binary.header.machine_type == lief.ELF.ARCH.x86_64:
         return cast(int, capstone.CS_ARCH_X86)
+    elif binary.header.machine_type == lief.ELF.ARCH.RISCV:
```

Comment:
Change should really be what is in this file + the examples.

Reply:
Should I add the case for RISC-V 32 bit? I think that capstone and LIEF support it.

Reply:
sure if you can test it. :)

The Makefile examples are used in the GitHub CI

Reply:
Yeah I tried to run the tests locally but they failed because I am on NixOS and I don't have binaries like `/bin/ls` and `/usr/bin/ruby`.

The only "test" I did was executing the RISC-V ELF in QEMU and running a few queries on it.